code/game/world.dm

```diff
@@ -384,15 +377,23 @@ GLOBAL_VAR(restart_counter)
 
 	game_state = (CONFIG_GET(number/extreme_popcap) && players >= CONFIG_GET(number/extreme_popcap)) //tells the hub if we are full
 
-	if (!host && hostedby)
-		features += "hosted by <b>[hostedby]</b>"
-
-	if(length(features))
-		s += ": [jointext(features, ", ")]"
+	s += "Time: <b>[gameTimestamp("hh:mm")]</b><br>"
```

Comment:
```suggestion
	s += "Time: <b>[gameTimestamp("hh:mm")]:00</b><br>"
```

What do you all think about something like this? I always get fooled by the counter and think the round started a minute and a half ago, for example.

Reply:
I could put the seconds in for real, other servers have been doing that (and it would look too clean if it was always :00).